### DISCUSSION TITLE
ocr() got an unexpected keyword argument 'slice'

### DISCUSSION THREAD

**Discussion by w243943** [PaddlePaddle/PaddleOCR#12533] [Q&A]
当我使用切片操作时会出现以下报错：ocr() got an unexpected keyword argument 'slice'，我所使用的paddle版本是：2.3.2，pddleocr版本是：2.7.3，cuda版本是10.1，所使用的系统是linux。具体报错位置是：  results = ocr.ocr(img_path, cls=True, slice=slice)
TypeError: ocr() got an unexpected keyword argument 'slice'
当不使用slice时可以正常运行，使用slice不行，应该是这个ocr函数没有初始化slice

**Comment by SWHL** [comment]
拉取最新的PaddleOCR代码尝试一下呢？

**Reply by w243943** [reply]
我使用就是最新版的paadleocr呀，最新发布难道不是2.7版本的吗

**Reply by SWHL** [reply]
是的，拉取最新main分支代码，试试吧。现在好多bug都修复了，但是还没来得及发版

**Comment by GreatV** [comment]
slice是刚刚引入的参数，2.7版本还没有
